CHANGES.rst

```diff
@@ -17,6 +17,11 @@ New Features
 Bug Fixes
 ^^^^^^^^^
 
+- ``photutils.isophote``
+
+  - Fix a bug that ``build_ellipse_model`` falls into an infinite loop
+    when the pixel to fit is outside of the image. [#1038]
```

Comment:
```suggestion
    when the pixel to fit is outside of the image. [#1039]
```

The changelog should contain the PR number where the bug was fixed, not the issue number.